CMakeLists.txt

```diff
@@ -160,6 +160,8 @@ endif()
 # Set output for libopenblas
 set_target_properties( ${OpenBLAS_LIBNAME} PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 set_target_properties( ${OpenBLAS_LIBNAME} PROPERTIES LIBRARY_OUTPUT_NAME_DEBUG "${OpenBLAS_LIBNAME}_d")
+set_target_properties( ${OpenBLAS_LIBNAME} PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE)
```

Comment:
If this is needed, then that means `exports\gensymbol` is outdated

Reply:
`zpotrs_` was not exported, and I strongly suspect that most of the other routines weren't either. The question is whether this is preferable to `_gensymbol`. As it reduces maintenance, I'd say yes.

Reply:
What's the difference between this and line 156?

Reply:
Probably the fact that the compiler is not MSVC (it's clang).

Reply:
CMake sets `MSVC` when `clang` is targeting `MSVC`

Reply:
I don't know then. Obviously it made a difference though.

Reply:
To me the proposed changes to exports/gensymbol still look completely unrelated to the PR topic.

Reply:
And I see the question of running gensymbol vs setting WINDOWS_EXPORT_ALL_SYMBOLS was raised in https://github.com/xianyi/OpenBLAS/pull/1279 already, seemed one is not a full replacement for the other.